dali/ft/daftformat.ipp

```diff
@@ -355,6 +356,226 @@ protected:
     Linked<IFileIOStream> stream;
 };
 
+class DALIFT_API JsonSplitter : public CInterface, implements IPTreeNotifyEvent
+{
+public:
+    IMPLEMENT_IINTERFACE;
+
+    JsonSplitter(const FileFormat & format, IFileIOStream &stream) : headerLength(0), pathPos(0), tangent(0), rowDepth(0), rowStart(0), rowEnd(0), footerLength(0), newRowSet(true)
+    {
+        LOG(MCdebugProgressDetail, unknownJob, "JsonSplitter::JsonSplitter(format.type :'%s', rowPath:'%s')", format.getFileFormatTypeString(), format.rowTag.get());
+
+        size = stream.size();
+        const char *rowPath = format.rowTag;
+        while (*rowPath=='/')
+            rowPath++;
+        pathNodes.appendList(rowPath, "/");
```

Comment:
xml rowTag's can't have '/', because of that, I was treating '/' as the special character.  Basically for XML it's a tag, for JSON its a path.  EclWatch always makes sure JSON paths start with '/'.  I don't really like it but didn't have time to add a new "markupFormat' indicator... have to make sure it gets serialized / deserialized, etc.  I'll look at it again.